src/app/mail/mail-list/mail-list.component.ts

```diff
@@ -12,16 +12,20 @@ import { Store } from '@ngrx/store';
 import { getMails } from '../../store/selectors';
 import { GetMails, GetMailboxes, FinalLoading } from '../../store/actions';
 import { OpenPgpService } from '../../store/services/openpgp.service';
+import { OnDestroy, TakeUntilDestroy } from 'ngx-take-until-destroy';
 
 declare var openpgp;
-
+@TakeUntilDestroy()
 @Component({
   selector: 'app-mail-list',
   templateUrl: './mail-list.component.html',
   styleUrls: ['./mail-list.component.scss']
 })
 export class MailListComponent implements OnInit {
```

Comment:
@Sage-Brezinsky you added `@TakeUntilDestroy()` annotation but you forgot to implment `OnDestroy`

Reply:
@Sage-Brezinsky you still didn't put the `export class MailListComponent implements OnInit , OnDestroy`
`OnDestroy` is a lifecycle hook and should be implemented.